.github/workflows/build.yml

```diff
@@ -507,29 +507,15 @@ jobs:
           yarn test:integration
 
   job_browser_playwright_tests:
-    name: Playwright (${{ matrix.bundle }}) Tests
+    name: Browser Integration Tests (Shard ${{ matrix.shard }})
```

Comment:
Is this name not ambiguous with the "Old" browser integration tests? 🤔

Reply:
Eh maybe... Any recommendations on what to name them? ^^ Should we just keep the old name in case we ever go through with this change?